python/nwis_client/pyproject.toml

```diff
@@ -1,6 +1,46 @@
 [build-system]
+requires = ["setuptools>=70"]
 build-backend = "setuptools.build_meta"
-requires = [
-  "setuptools>=42",
-  "wheel",
+
+[project]
+name = "hydrotools.nwis_client"
+authors = [{ name = "Austin Raney", email = "aaraney@protonmail.com" }]
```

Comment:
It looks like the move to `pyproject.toml` makes it easier to have multiple authors. Would you like to share the authorship? You should remain first author, of course.